urbs/validation.py

```diff
@@ -125,3 +133,10 @@ def validate_input(data):
                 raise KeyError("All names in the column 'Site' in input "
                                "worksheet 'DSM' must be from the list of site "
                                "names specified in the worksheet 'Site'.")
+
+# report that variable costs may have error if used with CO2 minimization and DCPF
+def validate_dc_objective(data, objective):
+    if not data['transmission'].empty:
+        if 'reactance' in data['transmission'].keys():
+            if any(data['transmission']['reactance'] > 0) and (objective == 'CO2') and any(data['transmission']['var-cost'] > 0):
+                print("!!!!\nif the C02 is selected as objective function while modelling DC transmission lines, variable costs may be different \n!!!!")
```

Comment:
Please include an error message which does not need that many punctuation marks... 
Please ensure pep8 conformity.

Reply:
I agree, the punctuation marks were removed.